src/TextToSpeech.hs

```diff
@@ -51,12 +51,22 @@ espeak_contrib myDir lang tmp txt =
     l = case lang of
             Language s    -> s
 
+say :: Language -> FilePath -> String -> (String, [String])
+say lang tmp txt =
+   ("say", ["-o", tmp, "--data-format=LEF32@8000", "-v", l, txt])
+  where
+   l = case lang of
+           Language "en" -> "Alex"
+	   Language "de" -> "Anna"
+	   Language "fr" -> "Thomas"
+	   Language s    -> s
```

Comment:
macOS supports a pretty large list of languages, which you can view with `say -v '?'`. Unfortunately the i18n code can't be used as argument but only the voice name. So, my intention was to use the voice name at the language variable instead of the i18n code.

But I think, we just stick to the three languages for now. If someone needs another language, it should be easy to add.